src/resource_sqlite.cpp

```diff
@@ -31,11 +31,138 @@
 #include "resource_sqlite.h"
 #include "core/io/file_access.h"
 
-void SQLiteDatabase::set_data(const PackedByteArray &p_data) {
-    data = p_data;
+void SQLiteDatabase::_bind_methods() {
+    ClassDB::bind_method(D_METHOD("get_sqlite"), &SQLiteDatabase::get_sqlite);
+    ClassDB::bind_method(D_METHOD("create_table", "table_name", "columns"), &SQLiteDatabase::create_table);
```

Comment:
I noticed we only did a portion of an object relation mapping system.

1. query builder https://hexdocs.pm/ecto/Ecto.Query.html
2. validator https://hexdocs.pm/ecto/Ecto.Changeset.html

Reply:
These are a lot of work.

Reply:
Thats true. Those could be done later I hope, and extend the API, or people can use it as an advanced feature by building queries.